examples/color/named_colors.py

```diff
@@ -25,18 +25,18 @@
 
 n = len(sorted_names)
 ncols = 4
-nrows = n // ncols + 1
+nrows = n // ncols
 
-fig, ax = plt.subplots(figsize=(8, 5))
+fig, ax = plt.subplots(figsize=(12, 10))
```

Comment:
Note that no image on the page should be larger than 900 pixels in width (=9 inch @ the default 100dpi). Otherwise the image will overlap (or be downsampled).

Otherwise this is a great idea! :+1: